main.cpp

```diff
@@ -238,7 +253,7 @@ void House::openWindows(int openedWindows)
     }
 }
 
-bool House::closeWindow(House::Kitchen myKitchen)
+bool House::closeWindow(House::Kitchen& myKitchen)
```

Comment:
pass by `const reference` if you're not modifying the passed-in object
- [ ] revise